CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 * Fix the `launder` and `finalize` methods of the `float` schema field query builder.
 * Fix the `launder` and `finalize` methods of the `integer` schema field query builder.
+* Rollbacks modifications that changed the delete `permission` name in `archive`.
```

Comment:
If this was never released then we do not need to mention in the changelog when we undo it.

Reply:
Removed